public/js/components/DataPropertyField.vue

```diff
@@ -0,0 +1,30 @@
+<template>
+  <label v-bind:class="{ invalid: !isValid }">
+    Data Property (supports
+    <a href="http://jmespath.org/tutorial.html" target="_blank">JMESPath</a>)
```

Comment:
In _my_ opinion, links should almost never have `target="_blank"`. It should be the user’s decision whether a link is opened in a new window/tab or not. Adding this attribute takes this decision away from the user and I don’t see a good reason for that in this use case.

I would like to hear some opinions on this. :)

(I realize that this was present in the code before.)

Reply:
I personally don't mind having the link open in a new window here, but if we use `target="_blank"` then it should be accompanied by `rel="noopener"` for security reasons (explanation here: https://developers.google.com/web/tools/lighthouse/audits/noopener)

Reply:
I agree. I followed the precedent set by the "what's this?" link on the sticker card settings for the alt text. However, navigating away in this case will lose the user's unsaved changes.